ansible-scylla-node/tasks/common.yml

```diff
@@ -23,16 +23,14 @@
   block:
     - name: check for current raid
       shell: |
-        C=$(cat /proc/mdstat |grep md0| wc -w)
-        echo $((0 + C - 4))
-      register: present_raid_disks
+        lsblk | grep scylla | wc -l
```

Comment:
isn't it too wide?
shouldn't you check for whole path?

Reply:
Will do.

Reply:
Done.

Reply:
@tarzanek please, re-review.